build-system/tasks/extension-helpers.js

```diff
@@ -630,6 +665,13 @@ async function buildNpmBinaries(extDir, name, options) {
         ),
         outfile: 'web-component.js',
         wrapper: '',
+        ...(options.useBentoCore
```

Comment:
Nitpick: This seems like another opportunity to modify the object based on the parameter rather than spreading with a default empty object.

Reply:
updated